process/block/shardblock.go

```diff
@@ -1280,6 +1289,7 @@ func (sp *shardProcessor) updateState(headers []data.HeaderHandler, currentHeade
 		return
 	}
 
+	// TODO: set proper finalized header in outport
 	sp.setFinalizedHeaderHashInIndexer(currentHeaderHash)
```

Comment:
line 1279: 
```
        if sp.enableEpochsHandler.IsFlagEnabledInEpoch(common.AndromedaFlag, header.GetEpoch()) {
			continue
		}
```

means there is no need to set final every block from the last final to the current?